src/people/widgetViews/PhasePlannerView.tsx

```diff
@@ -147,8 +147,16 @@ const PhasePlannerView: React.FC = () => {
       version: 1
     };
 
+    const ticketPayload = {
+      metadata: {
+        source: 'websocket',
+        id: initialTicketData.uuid
```

Comment:
This isn't correct. 
We need metadata.id = the websocket session ID

This is setup on page load of phasePlanner and logged out in the code if you check the ticket.